### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by trusli**
Added the primary bug that was fixed in 3.7.1

### CODE REVIEW

**Comment by trusli** [comment]
corrected spelling with PR #119

File: docs/release_notes.md [hunk 1]
```diff
@@ -64,6 +67,7 @@ AppHub
 
 * Continued to refine application templates in AppHub (introduced in RTS 3.6.0).  
 
+
```
Comment (vrozov) [inline docs/release_notes.md]: Revert back

File: docs/release_notes.md [hunk 2]
```diff
@@ -22,6 +23,8 @@ Other issues that are also fixed:
 * [SPOI-10698] - Allow Custom Properties with Config XML file while launching an Application
 * [SPOI-10737] - AppDataTracker application relaunches continuously and fails in a kerberized cluster.
 * [SPOI-10932] - Installation Wizard does not allow to complete gateway configuration
+* [SPOI-10773] - Application fails to run in fully enabled kerberized mode (DataTorrent Apex Jira reference)
```
Comment (vrozov) [inline docs/release_notes.md]: Please replace (DataTorrent Apex Jira reference) with link to APEXCORE-616 JIRA and remove the next line.

File: docs/release_notes.md (hunk 1 above)
Comment (trusli) [inline docs/release_notes.md]: Done

**Comment by vrozov** [comment]
Please squash commits

**Comment by trusli** [comment]
@vrozov squash done

**Pull request merged**